Simplified/LibraryFinder/UI/NYPLLibraryFinderLibraryCell.swift

```diff
@@ -0,0 +1,203 @@
+      verticalFittingPriority: .fittingSizeLevel
+    )
+    
+    return size
+  }
+  
+  // MARK: - UI
+  
+  func configureCell(type: NYPLLibraryFinderLibraryCellType, account: Account) {
+    self.type = type
+    configureBorder(for: type)
+    
+    nameLabel.text = account.name
+    // TODO: iOS-36 Assign data value to UI componenets after parser and data model are updated
+    descriptionLabel.text = account.subtitle
+    distanceLabel.text = "20 Mi"
+    postalCodeLabel.text = "12345"
+    
+    configureBadgeLabel(text: "BADGE")
+  }
+  
+  private func setupUI() {
+    backgroundColor = NYPLLibraryFinderConfiguration.cellBackgroundColor
+    layer.cornerRadius = NYPLLibraryFinderConfiguration.cellCornerRadius(type: self.type)
+    
+    let leftStackView = UIStackView(arrangedSubviews: [nameLabel, descriptionLabel])
+    leftStackView.axis = .vertical
+    leftStackView.spacing = stackViewSpacing
+    leftStackView.alignment = .leading
+    
+    let rightStackView = UIStackView(arrangedSubviews: [distanceLabel, postalCodeLabel, UIView()])
+    rightStackView.axis = .vertical
+    rightStackView.spacing = stackViewSpacing
+    rightStackView.alignment = .trailing
+    
+    addSubview(divider)
+    addSubview(leftStackView)
+    addSubview(rightStackView)
+    addSubview(badgeLabel)
+    
+    divider.autoPinEdgesToSuperviewEdges(with: .init(top: 0, left: 15, bottom: 0, right: 15), excludingEdge: .bottom)
+    divider.heightAnchor.constraint(equalToConstant: NYPLLibraryFinderConfiguration.borderWidth).isActive = true
+    
+    leftStackView.autoPinEdge(.top, to: .bottom, of: divider, withOffset: 20)
+    leftStackView.autoPinEdge(toSuperviewEdge: .leading, withInset: 15)
+    leftStackView.autoPinEdge(toSuperviewEdge: .bottom, withInset: 20)
+    
+    badgeLabel.autoPinEdge(.leading, to: .trailing, of: leftStackView, withOffset: 20)
+    badgeLabel.autoPinEdge(.top, to: .bottom, of: divider, withOffset: 20)
+    badgeLabel.autoPinEdge(toSuperviewEdge: .trailing, withInset: 15)
+    
+    badgeLabelWidthConstraint = badgeLabel.widthAnchor.constraint(greaterThanOrEqualToConstant: 60)
+    badgeLabelWidthConstraint?.isActive = true
+    badgeLabelHeightConstraint = badgeLabel.heightAnchor.constraint(equalToConstant: 20)
+    badgeLabelHeightConstraint?.isActive = true
+    badgeLabel.layer.cornerRadius = 10
+    
+    rightStackView.autoPinEdge(.top, to: .bottom, of: badgeLabel)
+    rightStackView.autoPinEdge(toSuperviewEdge: .trailing, withInset: 15)
+    rightStackView.autoPinEdge(toSuperviewEdge: .bottom, withInset: 10)
+    rightStackView.autoPinEdge(.leading, to: .trailing, of: leftStackView, withOffset: 20)
+    rightStackView.widthAnchor.constraint(greaterThanOrEqualToConstant: 60).isActive = true
+  }
+  
+  private func configureBadgeLabel(text: String?) {
+    if let text = text {
+      badgeLabel.text = text
+      let paddedWidth = badgeLabel.intrinsicContentSize.width + 2 * 10
```

Comment:
It's not clear if you intended it to be `(badgeLabel.intrinsicContentSize.width + 2) * 10` or `badgeLabel.intrinsicContentSize.width  + (2 * 10)`.

As of now, with current swift operator precedence it becomes the later which I don't think you intended.